openedx/core/djangoapps/schedules/signals.py

```diff
@@ -196,7 +196,9 @@ def _create_schedule(enrollment, enrollment_created):
 
     schedule = Schedule.objects.create(
         enrollment=enrollment,
+        # TODO remove 'start' field in removing writes from old field step in column renaming release
         start=content_availability_date,
+        start_date=content_availability_date,
```

Comment:
Oops, didn't see this!  Sorry!

Reply:
Okay, so it looks like this is called from the post_save signal on `CourseEnrollment`, not Schedule.  Are you sure there are no writes directly from the ScheduleAdmin? https://github.com/edx/edx-platform/blob/fb9d50ebcf96a1493779af5058489916c711aa14/openedx/core/djangoapps/schedules/admin.py#L124